File: KuantumJAVA/AntiMadde.java
public class AntiMadde extends KuantumNesnesi implements IKritik {

    public AntiMadde(String ID, double stabilite, int tehlikeSeviyesi) {
        super(ID, stabilite, tehlikeSeviyesi);
    }

    @Override
    public void AnalizEt() {
        setStabilite(getStabilite() - 25);
        System.out.println("Evrenin dokusu titriyor...");
    }

    @Override
    public void AcilDurumSogutmasi() {
        setStabilite(getStabilite() + 50);
        System.out.println("Antimadde soğutuldu!");
    }
}

    

File: KuantumJS2/AntiMadde.js
const KuantumNesnesi = require("./KuantumNesnesi");
const IKritik = require("./IKritik");

class AntiMadde extends KuantumNesnesi {
    constructor(ID, stabilite, tehlikeSeviyesi) {
        super(ID, stabilite, tehlikeSeviyesi);
    }

    analizEt() {
        this.setStabilite(this.getStabilite() - 25);
        console.log("Evrenin dokusu titriyor...");
    }

    acilDurumSogutmasi() {
        this.setStabilite(this.getStabilite() + 50);
        console.log("Antimadde soğutuldu!");
    }
}

module.exports = AntiMadde;
